aodndata/moorings/products_handler.py

```diff
@@ -126,6 +128,14 @@ def _read_manifest(self):
             raise InvalidFileContentError(
                 "manifest file '{self.input_file}' missing information (site_code, variables)".format(self=self)
             )
+        if 'products' in manifest:
```

Comment:
This if and the try statement above could be refactored into a validation function ```validate_manifest(manifest)```.

Reply:
It could. For now I don't think it's worth it. "products" is optional, and the other two keys are already being validated above.